src/google/protobuf/compiler/csharp/csharp_enum.cc

```diff
@@ -49,8 +49,8 @@ namespace protobuf {
 namespace compiler {
 namespace csharp {
 
-EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor) :
-    SourceGeneratorBase(descriptor->file()),
+EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor, const Options* options) :
```

Comment:
Looks like this could be a `const Options& options` (it's never going to be null) but I understand the desire to be consistent with the descriptor parameter. We might want to do a thorough cleanup at some point...